### PR TITLE
IGVF-1035-prediction-set-model-set

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

File: src/igvfd/schemas/changelogs/model.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`model.json`*
 
+### Minor changes since schema version 3
+
+* Make required properties `lab` and `award` not submittable. No further submission of models are accepted, please submit model sets instead.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/model.md:5]: please make the changelog more terse.
lab and award are still submittable, although by admins only.
I would go with: 

* Restrict `lab` and `award` to admins only.

File: src/igvfd/schemas/changelogs/prediction.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`prediction.json`*
 
+### Minor changes since schema version 3
+
+* Make required properties `lab` and `award` not submittable. No further submission of predictions are accepted, please submit prediction sets instead.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/prediction.md:5]: * Restrict `lab` and `award` to admins only.

File: src/igvfd/schemas/model.json [hunk 3]
```diff
@@ -62,6 +62,12 @@
         "schema_version": {
             "default": "3"
         },
+        "award": {
```
Comment (gabdank) [inline src/igvfd/schemas/model.json:65]: add comment explaining the restriction, on the way to deprecate use of the schemas

File: src/igvfd/schemas/prediction.json [hunk 4]
```diff
@@ -48,6 +48,12 @@
         "schema_version": {
             "default": "3"
         },
+        "award": {
```
Comment (gabdank) [inline src/igvfd/schemas/prediction.json:51]: add comment explaining that restriction is in place to drive towards deprecation ofthe schema

**Review by gabdank — CHANGES_REQUESTED** [review]

File: src/igvfd/schemas/changelogs/model.md (hunk 1 above)
Comment (ian-whaling) [reply src/igvfd/schemas/changelogs/model.md]: Added.

File: src/igvfd/schemas/changelogs/prediction.md (hunk 2 above)
Comment (ian-whaling) [reply src/igvfd/schemas/changelogs/prediction.md]: Added.

File: src/igvfd/schemas/model.json (hunk 3 above)
Comment (ian-whaling) [reply src/igvfd/schemas/model.json:65]: Added.

File: src/igvfd/schemas/prediction.json (hunk 4 above)
Comment (ian-whaling) [reply src/igvfd/schemas/prediction.json:51]: Added.

**Review by gabdank — APPROVED** [review]

**Pull request closed without merging**